binary/proto/scan_result.proto

```diff
@@ -646,7 +646,9 @@ message SecretData {
     GrokXAIAPIKey grok_xai_api_key = 6;
     GrokXAIManagementAPIKey grok_xai_management_api_key = 7;
     DockerHubPat docker_hub_pat = 8;
-    DigitalOceanAPIToken digitalocean = 9;
+    PostmanAPIKey postman_api_key = 9;
+    PostmanCollectionAccessToken postman_collection_access_token = 10;
+    DigitalOceanAPIToken digitalocean =11;
```

Comment:
Please don't change existing message number since servers that store binary serialized versions of previous instances of the proto might depend on this number to know what secret type they're storing.